### ISSUE TITLE
Reduce the number of slides in the "short" decks

### ISSUE THREAD

**Issue by tclayson** [kennethkutyn-optimizely/deckbuilder#20]
There are a lot of slides and even the "short" decks seem to have a LOT of slides in them. We should try and be ruthless and reduce the amount of slides in the decks, especially the "short" decks (e.g. "Single Page Apps" seem to have 14 slides? Is this necessary?)

**Comment by kennethkutyn-optimizely** [comment]
agreed, or maybe we need to start letting people build their own deck builder. Is one-size-fits-all really feasible?

**Comment by kennethkutyn-optimizely** [comment]
oo, what if we add a query parameter with the SEname, then deckbuilder app check for query parameter and if it exists, then we can have a mapping file of SE name to master deck URL and Structure. Then any SE who wants to can manage their own master deck.

**Comment by tclayson** [comment]
This might be difficult to achieve while still making sure everyone's decks are up to date with the latest slides. I wonder if we can make some sort of "micro deck adjustment" functionality and allow people to create their own "templates" for each of the decks instead. Something to look into I think. :)
